### PR TITLE
Update copyright to include 2021.

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR contains code ducumentation updates
- It does the following:
  - Updates all copyright info to contain the year 2021
  - Adds copyright info to files where it was absent where I found them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+[comment]: # (#################################################################)
+[comment]: # (Copyright 2017-2120, Lawrence Livermore National Security, LLC)
```
Comment (kennyweiss) [inline README.md:2]: 2120 -> 2021

File: README.md [hunk 2]
```diff
@@ -45,7 +54,7 @@ Thanks to all of Axom's
 License
 -------
 
-Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC. 
+Copyright (c) 2017-2121, Lawrence Livermore National Security, LLC.
```
Comment (kennyweiss) [inline README.md:57]: 2121 -> 2021

File: RELEASE [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Axom: ................................, version 0.4.0
 
-Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC. 
+Copyright (c) 2017-2120, Lawrence Livermore National Security, LLC.
```
Comment (kennyweiss) [inline RELEASE:5]: 2120 -> 2021

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,3 +1,13 @@
+
+[comment]: # (#################################################################)
+[comment]: # (Copyright 2017-2120, Lawrence Livermore National Security, LLC)
```
Comment (kennyweiss) [inline RELEASE-NOTES.md:3]: 2120 -> 2021

File: scripts/update_copyright_date.sh [hunk 5]
```diff
@@ -53,9 +53,29 @@ for i in `cat files2change`
...
     cp $i $i.sed.bak
-    sed "s/Copyright (c) 2017-2019/Copyright (c) 2017-2020/" $i.sed.bak > $i
+    sed "s/Copyright (c) 2017-2020/Copyright (c) 2017-2021/" $i.sed.bak > $i
 done
 
+echo LICENSE
+cp LICENSE LICENSE.sed.bak
+sed "s/Copyright (c) 2017-2020/Copyright (c) 2017-2021/" LICENSE.sed.bak > LICENSE
+
+echo RELEASE
```
Comment (kennyweiss) [inline scripts/update_copyright_date.sh:63]: I think these lines either needs some tweaks, or an initial manual intervention

**Review by kennyweiss — CHANGES_REQUESTED** [review]
Thanks @rhornung67 -- I think your sed lines need a little tweaking for the custom files

File: README.md (hunk 1 above)
Comment (rhornung67) [reply README.md]: Thanks. I can't type.  I fixed this and a few others.

File: scripts/update_copyright_date.sh (hunk 5 above)
Comment (rhornung67) [reply scripts/update_copyright_date.sh:53]: @kennyweiss thanks. I think I fixed everything. I shouldn't do a PR while I'm in a meeting. Sorry.

File: README.md (hunk 2 above)
Comment (kennyweiss) [inline README.md:57]:
```suggestion
Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC. 
```

**Review by kennyweiss — APPROVED** [review]
No problem -- looks like you got all but one

File: README.md (hunk 2 above)
Comment (rhornung67) [reply README.md]: Got it. Thanks.

**Review by nselliott — APPROVED** [review]

**Pull request merged**